module/VuFind/src/VuFind/Db/Row/UserList.php

```diff
@@ -284,7 +287,7 @@ public function isPublic()
      *
      * @param \VuFind\Db\Row\User|bool $user  Logged-in user (false if none)
      * @param bool                     $force Should we force the delete without
-     * checking permissions?
+     *                                        checking permissions?
```

Comment:
This whitespace is not needed -- long doc comments should wrap to the left margin, they don't need to be aligned.